Lib/test/test_abstract_numbers.py

```diff
@@ -1,14 +1,32 @@
 """Unit tests for numbers.py."""
 
+import abc
 import math
 import operator
 import unittest
-from numbers import Complex, Real, Rational, Integral
+from numbers import Complex, Real, Rational, Integral, Number
+
+
+def concretize(cls):
+    def not_implemented(*args, **kwargs):
+        raise NotImplementedError()
+
+    for name in dir(cls):
+        try:
+            value = getattr(cls, name)
+            if value.__isabstractmethod__:
+                setattr(cls, name, not_implemented)
+        except AttributeError:
+            pass
+    abc.update_abstractmethods(cls)
+    return cls
+
 
 class TestNumbers(unittest.TestCase):
     def test_int(self):
         self.assertTrue(issubclass(int, Integral))
         self.assertTrue(issubclass(int, Complex))
+        self.assertTrue(issubclass(int, Number))
```

Comment:
Test also with Rationale and Real. Test all concrete types with all 5 abstract types.

Reply:
Please add also similar checks for other concrete types (test_float, test_complex).

Reply:
Sorry, I forgot to check these, updated!